src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");

```

Comment:
The message 'Excavator started working' could be more informative. According to the checklist, it should indicate what type of machine is working. Consider specifying the type of machine in a more descriptive way, for example, 'Excavator is now digging the ground.'